packages/calypso-e2e/test/browser-helper.test.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import { describe, expect, test, beforeAll, afterAll } from '@jest/globals';
+
+/**
+ * Internal dependencies
+ */
+import { getViewportName, getLocale, getViewportSize } from '../src/browser-helper';
+
+let env: NodeJS.ProcessEnv;
+
+beforeAll( function () {
+	env = process.env;
+} );
+
+describe( 'BrowserHelper Tests', function () {
```

Comment:
This external `describe` is not doing anything, I'd consider deleting it.

Or moving the `beforeAll` and `afterAll` hooks inside.

Reply:
I had thought this external hook was acting to limit the number of times the hooks run to 1 per file. 

With that said, I will try moving the hooks.